pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/QueryRunnerTestBase.java

```diff
@@ -181,6 +182,9 @@ protected void compareRowEquals(List<Object[]> resultRows, List<Object[]> expect
         }
         return Double.compare(ld, rd);
       } else if (l instanceof String) {
+        if (r instanceof byte[]) {
```

Comment:
When querying bytes column, left side from pinot is a hex string, right side from h2 is byte[]

Reply:
ah. got it... so we want to make it hex string. that sound like a good plan

Reply:
postgres give hex value, but with with a prefix: `\x`, e.g. `\x4A6F686E20446F65`